FrameworksAndDrivers/Web/DotLanches.Api/Dtos/PagamentoResponseDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DotLanches.Api.Dtos;
+
+public class PagamentoResponseDto
+{
+    public int IdPedido { get; set; }
+
+    public bool PaymentStatus { get; set; }
```

Comment:
Como esse campo só tem duas opções, não sei se seria um status, penso que na verdade ele representa se foi aceito ou não. Então acho que poderia ser:
```suggestion
    public bool Accepted{ get; set; }
```